src/player.py

```diff
@@ -4,7 +4,7 @@ class Player(object):
     """Describe the player, his inventory, traits, position and history"""
 
     def __init__(self, state):
-        pass
```

Comment:
state = None
self.state = state

Reply:
If you merge I don't think you need to recommit this change